types/openlayers/index.d.ts

```diff
@@ -7597,7 +7599,7 @@ declare module ol {
          * @observable
          * @api stable
          */
-        setPosition(position: (ol.Coordinate)): void;
+        setPosition(position: (ol.Coordinate) | undefined): void;
```

Comment:
The "| undefined" should be inside the bracket to be coherent with the styling of the file.